server/src/main/java/com/genymobile/scrcpy/wrappers/ClipboardManager.java

```diff
@@ -1,270 +1,43 @@
 package com.genymobile.scrcpy.wrappers;
 
-import com.genymobile.scrcpy.AndroidVersions;
 import com.genymobile.scrcpy.FakeContext;
-import com.genymobile.scrcpy.util.Ln;
 
 import android.content.ClipData;
-import android.content.IOnPrimaryClipChangedListener;
-import android.os.Build;
-import android.os.IInterface;
-
-import java.lang.reflect.Method;
 
 public final class ClipboardManager {
-    private final IInterface manager;
-    private Method getPrimaryClipMethod;
-    private Method setPrimaryClipMethod;
-    private Method addPrimaryClipChangedListener;
-    private int getMethodVersion;
-    private int setMethodVersion;
-    private int addListenerMethodVersion;
+    private final android.content.ClipboardManager manager;
 
     static ClipboardManager create() {
-        IInterface clipboard = ServiceManager.getService("clipboard", "android.content.IClipboard");
-        if (clipboard == null) {
+        android.content.ClipboardManager manager = (android.content.ClipboardManager) FakeContext.get()
+                .getSystemService(FakeContext.CLIPBOARD_SERVICE);
+        if (manager == null) {
```

Comment:
`ClipboardManager`'s constructor will throw when `ClipboardService` is not found: https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/core/java/android/content/ClipboardManager.java;l=134-135;drc=16f7cdf2e01e158efe8d8334a15e9e20e59de929